src/libaktualizr/package_manager/ostreemanager.cc

```diff
@@ -70,6 +70,8 @@ data::InstallationResult OstreeManager::pull(const boost::filesystem::path &sysr
                                              OstreeProgressCb progress_cb) {
   const std::string refhash = target.sha256Hash();
   const char *const commit_ids[] = {refhash.c_str()};
+  const std::string branchname = target.filename().substr(0, target.filename().rfind("-"));
+  const char *const branchnames[] = {branchname.c_str()};
```

Comment:
rfind can return `string::npos`. Should we detect that and not try and set a branch if that's the case?